connections/.devcontainer/post-install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+# Convenience workspace directory for later use
+WORKSPACE_DIR=$(pwd)
+
+# install all ACA-Py requirements
+python -m pip install --upgrade pip
+
+# install black for formatting
+pip3 install black
```

Comment:
I need to update this from plugins global but this doesn't need to be install anymore.